test/passes/fannkuch3_manyopts_dwarf.bin.txt

```diff
@@ -2303,7 +2303,7 @@ Contains section .debug_info (851 bytes)
 Contains section .debug_loc (1073 bytes)
 Contains section .debug_ranges (88 bytes)
 Contains section .debug_abbrev (333 bytes)
-Contains section .debug_line (2682 bytes)
+Contains section .debug_line (2642 bytes)
```

Comment:
Is this expected and accounted for?

Reply:
I believe so, yes. StackIR saves a few bytes here and there, which allows encoded offsets to be a bit shorter in some cases. Any increase in size (in this or any other field) would have been very suspicious, however. A large decrease would also have been odd.

I did also run `llvm-dwarfdump --validate` on the output and it is equally valid as before this PR (which is, 99% valid but for one known pre-existing minor issue).